Jenkinsfile

```diff
@@ -68,70 +68,105 @@ pipeline {
         }
       }
-
     }
 
-    stage('Integration Test') {
-      steps {
-        // Run a simple webserver serving the dist folder statically
-        // before we run the cypress tests
-        writeFile file: 'integration-tests.sh', text: [
-          'export PATH=`pwd`/node_modules/.bin:$PATH',
-          'http-server -p 4200 dist&',
-          'SERVER_PID=$!',
-          './scripts/ci/run-integration-tests',
-          'RET=$?',
-          'echo "cypress exit status: ${RET}"',
-          'sleep 10',
-          'echo "kill server"',
-          'kill $SERVER_PID',
-          'exit $RET'
-        ].join('\n')
-
-        unstash 'dist'
+    // stage('Integration Test') {
+    //   steps {
+    //     // Run a simple webserver serving the dist folder statically
+    //     // before we run the cypress tests
+    //     writeFile file: 'integration-tests.sh', text: [
+    //       'export PATH=`pwd`/node_modules/.bin:$PATH',
+    //       'http-server -p 4200 dist&',
+    //       'SERVER_PID=$!',
+    //       './scripts/ci/run-integration-tests',
+    //       'RET=$?',
+    //       'echo "cypress exit status: ${RET}"',
+    //       'sleep 10',
+    //       'echo "kill server"',
+    //       'kill $SERVER_PID',
+    //       'exit $RET'
+    //     ].join('\n')
+
+    //     unstash 'dist'
+
+    //     ansiColor('xterm') {
+    //       retry(2) {
+    //         sh '''bash integration-tests.sh'''
+    //       }
+    //     }
+    //   }
+
+    //   post {
+    //     always {
+    //       archiveArtifacts 'cypress/**/*'
+    //       junit 'cypress/results.xml'
+    //     }
+    //   }
+    // }
+
+    // stage('System Test') {
+    //  steps {
+    //    withCredentials([
+    //       [
+    //         $class: 'AmazonWebServicesCredentialsBinding',
+    //         credentialsId: 'f40eebe0-f9aa-4336-b460-b2c4d7876fde',
+    //         accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+    //         secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+    //       ]
+    //     ]) {
+    //      unstash 'dist'
+
+    //      ansiColor('xterm') {
+    //        retry(2) {
+    //          sh '''dcos-system-test-driver -j1 -v ./system-tests/driver-config/jenkins.sh'''
+    //        }
+    //      }
+    //    }
+    //  }
+
+    //  post {
+    //    always {
+    //      archiveArtifacts 'results/**/*'
+    //      junit 'results/results.xml'
+    //    }
+    //  }
+    // }
+
+    // when env.BRANCH_NAME == 'master', we will update the mesosphere:dcos-ui/latest branch
+    // when env.BRANCH_NAME =~ /(tag-name-regex)/, we will open a release PR against dcos/dcos
+    stage('Upload Build') {
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ /^v?(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?/ || env.BRANCH_NAME == 'bill/feat/dcos-ui-cd' || env.BRANCH_NAME == 'test-dcos-ui-cd2' }
```

Comment:
Might be a bit esoteric, but couldn't I fork this project and run a PR from my master which then executes the upload build script? Maybe we can add a small follow-up Jira to mitigate the risk here or am I on the wrong track?